tests/python/unittest/test_text.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# 'License'); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+from collections import Counter
+import os
+
+from mxnet.test_utils import *
+from mxnet.text import utils as tu
+from mxnet.text import glossary as glos
+
+
+def _get_test_str_of_tokens(token_delim, seq_delim):
+    seq1 = token_delim + token_delim.join(['Life', 'is', 'great', '!']) \
+           + token_delim + seq_delim
+    seq2 = token_delim + token_delim.join(['life', 'is', 'good', '.']) \
+           + token_delim + seq_delim
+    seq3 = token_delim + token_delim.join(['life', "isn't", 'bad', '.']) \
+           + token_delim + seq_delim
+    seqs = seq1 + seq2 + seq3
+    return seqs
+
+
+def _mk_dir_of_files(path, token_delim, seq_delim):
+    if not os.path.exists(path):
+        os.makedirs(path)
+    seqs = _get_test_str_of_tokens(token_delim, seq_delim)
+
+    with open(os.path.join(path, '1.txt'), 'w') as fout:
+        fout.write(seqs)
```

Comment:
Please try to mock all file operations in unit tests.  See https://docs.python.org/3/library/unittest.mock.html#mock-open.

Reply:
I tried to be consistent with our existing unit tests:
https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_image.py#L108